js/content/store.js

```diff
@@ -64,22 +65,19 @@ class Customizer {
         for (let element of elements) {
             element.classList.toggle("esi-shown", state);
             element.classList.toggle("esi-hidden", !state);
-            element.classList.add("esi-customizer"); // for dynamic entries on home page
+            element.classList.add("esi-customizer");
             element.dataset.es_name = name;
             element.dataset.es_text = text;
         }
 
         return this;
-    };
-
-    addDynamic(titleNode, targetNode) {
-        let textValue = this._textValue(titleNode);
+    }
 
-        console.warn("Node with textValue %s is not recognized!", textValue);
-        let option = textValue.toLowerCase().replace(/[^a-z]*/g, "");
-        if (option === "") { return; }
+    addDynamic(node) {
+        let text = this._textValue(node).toLowerCase();
+        if (text === "") return;
 
-        this.add("dynamic_" + option, targetNode, textValue);
+        this.add(`dynamic_${text}`, node, text);
```

Comment:
Since the text here is translated by Steam, the options won't carry over if users switch languages, will this be a problem?

Reply:
I can't think of a better solution right now